conf/core/AttributeConflation.conf

```diff
@@ -7,6 +7,8 @@
   "conflate.post.ops": "$(conflate.post.ops);hoot::RemoveElementsVisitor;hoot::RemoveUnknown2Visitor",
   "duplicate.name.preserve.original.name": "true",
   "highway.merge.tags.only": "true",
+  "reader.conflate.use.data.source.ids.1": "false",
+  "reader.conflate.use.data.source.ids.2": "true",
```

Comment:
This swap for attribute conflation shouldn't really do much right now but in the future when we try to keep IDs with attribute conflation it will already be setup.